src/c++/perf_analyzer/test_load_manager_base.h

```diff
@@ -103,29 +127,32 @@ class TestLoadManagerBase {
     // there are never any overlapping requests -- they always immediately exit
     //
     if (params_.sequence_length != 1) {
-      CHECK(
-          params_.num_of_sequences ==
-          stats->sequence_status.max_live_seq_count);
+      expected_num_seq = std::min(expected_num_seq, params_.sequence_id_range);
```

Comment:
Why is `params_.sequence_id_range` involved here?

Reply:
sequence_id_range is the number of valid sequence IDs that are allowed. If the valid range of sequence IDs is 1 to 5, it doesn't matter if you say you want 20 sequences, you're only going to get 5.